docs/source/general/guides/drift_guide.rst

```diff
@@ -181,6 +185,32 @@ For code examples, see `here <#tabular-checks>`__
 All of these checks appear also in the `deepchecks interactive demo <https://checks-demo.deepchecks.com>`__, where you can
 insert corruption into the data and see the checks at work.
 
+Text (NLP) Data
+---------------
+
+Regarding `data <#data-drift>`__ or `concept drift <#concept-drift>`__
+In text data, we can't measure drift on the text directly, as text is not structured data that can be measured.
+However, we can use different methods to represent the text as a structured variable, and then measure drift on that variable.
+In deepchecks-NLP, we use 2 such methods: :ref:`Text Embeddings <nlp__embeddings_guide>` and :ref:`Text Properties <nlp__properties_guide>`.
```

Comment:
```suggestion
In deepchecks-nlp, we use 2 such methods: 

- :ref:`Text Embeddings <nlp__embeddings_guide>`
- :ref:`Text Properties <nlp__properties_guide>`.
```

Reply:
I think I'm against it - as there are similar bullets after the next paragraph, may seem unnecessarily long